frontend/shared/src/components/NotFound/__tests__/NotFound.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { render, screen } from '@testing-library/react';
 
-import NotFound from '../NotFound';
+import { NotFound } from 'components/NotFound';
```

Comment:
For unit tests, it is better to leave a relative import to the function/component that is tested. This will allow the containing directory to be moved elsewhere without having to change its code

Reply:
That's right, I changed it to test the resolution and forgot to change